backend/frontend/templates/frontend/auth/page_permission_denied.html

```diff
@@ -0,0 +1,22 @@
+{% extends "frontend/user/base.html" %}
```

Comment:
- The task said to extend the `base_user_logged_in` template. It also makes little sense to extend the base user because what happens if that user base template drastically changes?